front-end/src/services/Civic.js

```diff
@@ -24,7 +27,20 @@ export default class Civic {
      * @param {string} password - password
      * @returns {AccountExtended}
      */
-    async accountLogin(accountName, password, commonName) { };
+    async accountLogin(accountName, password, commonName) {
+        const privKey = ecc.seedPrivate(crypto.createHash("sha256").update(accountName + password).digest("hex"))
+        const pubKey = ecc.privateToPublic(privKey)
+
+        const response = await Api.post('login', {
+            accountName,
+            pubKey
+        })
+
+        this.account = response.data
+        await Contract.initializeContract()
```

Comment:
this line fails
this.civicContract.initializeContract()

Reply:
Will investigate further tonight.